tests/unit_tests/storage/migration/test_version_3.py

```diff
@@ -19,7 +19,10 @@ def test_migrate_observations(setup_case, set_ert_config):
     with open_storage(ert_config.ens_path, "w") as storage:
         assert len(list(storage.experiments)) == 1
         experiment = list(storage.experiments)[0]
-        assert experiment.observations == ert_config.observations
+        assert experiment.observations == {
```

Comment:
Shouldnt:
```python
assert experiment.observations == ert_config.observations.datasets 
```
also work?

Reply:
Hmm, it probably should yep.. Does not at the moment but that might be an issue with the migration. Looking into it

Reply:
Was a bug in `to4` migration, should now be fixed